knative-kubecon/eventing-install.txt

```diff
@@ -0,0 +1,104 @@
+minishift config set image-caching true
+# Add new user called admin with password with role cluster-admin
+minishift addons enable admin-user
+# Allow the containers to be run with any uid
+minishift addons enable anyuid
+# Start minishift
+minishift start
+
+#Redo each time
+
+# Enable admission controller webhooks
+# The configuration stanzas below look weird and are just to workaround
+# https://bugzilla.redhat.com/show_bug.cgi?id=1635918
+minishift openshift config set --target=kube --patch '{
+    "admissionConfig": {
+        "pluginConfig": {
+            "ValidatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
+                    "kind": "WebhookAdmission",
+                    "kubeConfigFile": "/dev/null"
+                }
+            },
+            "MutatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
+                    "kind": "WebhookAdmission",
+                    "kubeConfigFile": "/dev/null"
+                }
+            }
+        }
+    }
+}'
+ 
+eval $(minishift oc-env)
+until oc login -u admin -p admin; do sleep 5; done;
+ 
+# Setup the project myproject for Knative use
+oc project myproject
+oc adm policy add-scc-to-user privileged -z default
+
+# Addons
+git clone https://github.com/openshift-cloud-functions/minishift-addons.git
+minishift addons install minishift-addons/istio
+minishift addons install minishift-addons/knative
+minishift addons install minishift-addons/knative-eventing #Installs SHA 80860ba
+
+minishift addons apply istio  
+minishift addons apply knative 
+minishift addons apply knative-eventing
+
+# Only for macOS
+sudo route -n add -net $(minishift openshift config view | grep ingressIPNetworkCIDR | awk '{print $NF}') $(minishift ip)
+# Only for Linux
+sudo ip route replace $(minishift openshift config view | grep ingressIPNetworkCIDR | sed 's/\r$//' | awk '{print $NF}') via $(minishift ip)
+  
+# Set the address of the knative gateway
+export IP_ADDRESS=$(oc get svc knative-ingressgateway -n istio-system -o 'jsonpath={.status.loadBalancer.ingress[0].ip}')
+
+#Example
+oc adm policy add-cluster-role-to-user cluster-admin -z default -n myproject
+
+#EventSource, move to myproject
+curl -L https://storage.googleapis.com/knative-releases/eventing/latest/release-source-k8sevents.yaml \
+  | sed 's/namespace: .*/namespace: myproject/' \
+  | oc -n myproject apply -f -
+
+## ADM fu...
+oc adm policy add-scc-to-user anyuid -z feed-sa -n myproject
+oc adm policy add-scc-to-user privileged -z feed-sa -n myproject
+oc adm policy add-scc-to-user anyuid -z default -n myproject
+oc adm policy add-scc-to-user privileged -z default -n myproject
+
+## install demo but change namespace to myproject
+curl -L https://raw.githubusercontent.com/knative/docs/master/eventing/samples/k8s-events/serviceaccount.yaml \
```

Comment:
These URLs are now returning 404 since the k8s-events folder was deleted and replaced by the new kubernetes-event-source

Reply:
@markito here they are: https://github.com/openshift-cloud-functions/demos/pull/2